recipes/pygame/meta.yaml

```diff
@@ -44,6 +44,7 @@ requirements:
     - freetype
     - jpeg
     - libpng
+    - zlib
     - libzlib
```

Comment:
If pygame genuinely needs zlib, then just use `- zlib` (i.e. remove `libzlib`). If it _shouldn't_ depend on zlib, then also remove `libzlib` but add a comment.

Context: basically, `libzlib` by itself is useless because it has no metadata, and should only come in through the run-export of `zlib` as a host-dep; because then library is installed correctly (for the needs of the respective package) and doesn't need to be found again.

@hmaarrfk can correct me if I'm saying something silly

Reply:
Yes, it needs zlib as that contains the pkg-config file that is needed on Windows.